package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "s54_food_places",
+  "version": "1.0.0",
+  "description": "Website where People can explore and help people explore different food places in Punjab.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Update the license field with the correct license for the project. This provides information about how others can use and contribute to your project. [medium]